src/main/java/com/mera/numbers/service/RecordService.java

```diff
@@ -28,6 +34,18 @@ public void printRecord(Record record) {
   }
 
   public void processRecord() {
-    printRecord(getRecordFromApi());
+    saveRecord(getRecordFromApi());
```

Comment:
You should print records to console by the logger and after that save them to database.
Use @Slf4j annotation to initialize logger